test_files/file_comment/comment_with_no_newline.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+/**
+ * @fileoverview
+ * @suppress {missingRequire}
+ */

```

Comment:
I think we use tslint to disallow this kind of input (with the missing newline), don't we?